ui/imports/shared/popups/EnableBiometricsPopup.qml

```diff
@@ -0,0 +1,65 @@
+import QtQuick
+import QtQuick.Layouts
+
+import StatusQ.Core
+import StatusQ.Core.Theme
+import StatusQ.Popups.Dialog
+import StatusQ.Controls
+import StatusQ.Components
+
+StatusDialog {
+    id: root
+
+    property string errorText
+    property bool loading
+
+    signal enableBiometricsRequested()
+
+    implicitWidth: 480
+    title: qsTr("Enable biometrics")
+
+    ColumnLayout {
+        anchors.fill: parent
+        spacing: Theme.bigPadding
+
+        StatusImage {
+            Layout.preferredWidth: 270
+            Layout.preferredHeight: 260
+            Layout.alignment: Qt.AlignHCenter
+            mipmap: true
+            smooth: false
+            source: Theme.png("onboarding/enable_biometrics")
+        }
+        StatusBaseText {
+            Layout.fillWidth: true
+            text: root.errorText
+            color: Theme.palette.dangerColor1
+            wrapMode: Text.WordWrap
+            horizontalAlignment: Text.AlignHCenter
+        }
+        StatusBaseText {
+            Layout.fillWidth: true
+            text: qsTr("Would you like to enable biometrics to fill in your password? You will use biometrics for signing in to Status and for signing transactions.")
+            color: Theme.palette.baseColor1
+            wrapMode: Text.WordWrap
+            horizontalAlignment: Text.AlignHCenter
+        }
+    }
+
+    footer: StatusDialogFooter {
+        dropShadowEnabled: true
+        rightButtons: ObjectModel {
```

Comment:
how about splitting the buttons. Both buttons on the right side looks kind of odd

Reply:
![IMG_755E8F314060-1](https://github.com/user-attachments/assets/da790d6d-db24-4e96-87f0-cf9f1c027c46)
 this doesnt look good :D

Reply:
ohh, still a bit odd 😄 Sorry for the bad idea..
I think the problem was that both are primary buttons. Looking at the other popups I see we're not really splitting the actions.

![Screenshot_20251024_150904_Status](https://github.com/user-attachments/assets/447497f9-c1ba-4d96-a9b1-81856ea0b44c)